webrepl.html

```diff
@@ -17,6 +17,10 @@
     font: 20px/1.5 sans-serif;
   }
 
+  #outer {
+    position: fixed;
+  }
+
 /*
   .terminal {
```

Comment:
sorry I'm apparently no master in doint the github line comment :)

I meant the .terminal section